scripts/run_virtual_iridium.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+SERVER_URL=${1:-127.0.0.1:8000}
```

Comment:
Can we use a different port? I'm planning to add the docs site to sailbot_workspace, which uses port 8000

Reply:
It actually opened port 8080 for me. Web hook server endpoint is 8000, http server port is 8080

Reply:
Changed to 8081. Also clarified what the variables/arguments refer to.